doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -301,9 +305,12 @@ Where technically feasible the following system events must be recorded:
 * Successful and unsuccessful login attempts
```

Comment:
We should detail there are three points of 'login' and explain the methods used to record the login

1. Command line authentication
2. Horizon authentication
3. SSH authentication and sudo on the computes, controllers, storage nodes

Reply:
Opened Issue #1936